FreeRTOS-Plus/Demo/FreeRTOS_Plus_TCP_Echo_Qemu_mps2/syscalls.c

```diff
@@ -105,6 +98,11 @@ int _write( int file,
     return len;
 }
 
+extern unsigned long _heap_bottom;
```

Comment:
Why not keep these variables at the top?

Reply:
This was also about merging different ports to the same code, but I'll leave this now also just as is.
The other port had additional ifdef code for picolibs, so had additional pressure to change this.